externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSessionImpl.scala

```diff
@@ -39,6 +40,10 @@ class SparkSessionImpl(
     val spark: SparkSession)
   extends AbstractSession(protocol, user, password, ipAddress, conf, sessionManager) {
 
+  override val handle: SessionHandle =
+    conf.get(KyuubiReservedKeys.KYUUBI_SESSION_HANDLE_KEY).map(SessionHandle.fromUUID).getOrElse(
+      SessionHandle())
```

Comment:
`KYUUBI_SESSION_HANDLE_KEY` should always present
```suggestion
  override val handle: SessionHandle = SessionHandle.fromUUID(conf(KYUUBI_SESSION_HANDLE_KEY))
```